samples/productSearch/productSets.js

```diff
@@ -134,6 +134,45 @@ async function deleteProductSet(projectId, location, productSetId) {
   // [END vision_product_search_delete_product_set]
 }
 
+async function purgeProductsInProductSet(projectId, location, productSetId) {
```

Comment:
I think this file needs a little reformatting to fit the [updated sample policies](https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head) where we wrap everything in a main function

Reply:
Doesn't that style require that samples also be split into individual files rather than using yargs commands? I can def overhaul this library's samples to match the updated policies across the board in another PR though.

Reply:
^ ugh probably, which means this is a strong candidate for making an internal cleanup bug and leaving this as is

Reply:
@bcoe thisis the question I pinged you about - any opinions? (it's okay if not! )

Reply:
:wave: there are some nice aspects to splitting into multiple files, mainly that the READMEs generated suddenly get more entries (one for each sample). _but_, happy to compromise if we make sure we have a tracking issue labeled `process` :smile: hate to block a PR, if you don't have the cycles to split things apart right now.

Reply:
I'd "love" to see the `process` labeled issue. Then we can have someone port these to the generator to get all of the latest process hotness for free (instead of spending the cycles splitting things apart).

Let's git 'er done :)

Reply:
We got this issue open https://github.com/googleapis/nodejs-vision/issues/446